modules/ROOT/pages/_partials/commons/common-document.adoc

```diff
@@ -449,8 +461,17 @@ Document Changes::
 --
 [source, Kotlin, subs="attributes+"]
 ----
-
-include::android:example$snippets/app/src/main/kotlin/com/couchbase/code_snippets/FlowExamples.kt[tag="flow-as-document-change-listener"]
+fun docChangeFlowExample(collection: Collection, owner: String): LiveData<DocumentChange?> {
+        // end::flow-as-database-change-listener[]
```

Comment:
Same deal.  Let's fix the code to which this points and not get code where it can't be verified.